JustSaying/Messaging/INotificationSubscriber.cs

```diff
@@ -8,8 +9,7 @@ namespace JustSaying.Messaging
     public interface INotificationSubscriber : INotificationSubscriberInterrogation
     {
         void AddMessageHandler<T>(Func<IHandlerAsync<T>> handler) where T : Message;
-        void Listen();
-        void StopListening();
+        void Listen(CancellationToken cancellationToken);
```

Comment:
We'd need to document this and make it part of `7.0.0` as it's a breaking interface change.

Reply:
This could be an overload and the original passes `CancellationToken.None`, but I feel like it should probably go out with a major release anyway, if the fluent api gets updating for example.

Reply:
While you can use `CancellationToken.None` here, I would prefer that users are steered by the docs down the path of "graceful shutdown when the app exits" which means that the host app should use a `CancellationTokenSource`.